### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by githubfan1** [Keriew/augustus#834] [General]
I am wondering when would be the next official release. Any time soon? ty

**Comment by crudelios** [comment]
Considering the amount of pending stuff, the new release is still well off!

Don't expect it for the next six months up to a year.

**Reply by P2C2E1993** [reply]
Hi! I'm eagerly waiting for the new release, and was wondering if there have been any updates.

**Reply by crudelios** [reply]
There isn't much stuff left, but still quite a bit of hard to fix bugs and three or four new things missing.

I've been away since September which hasn't helped either.

**Reply by P2C2E1993** [reply]
Thanks for your response. I've been playing the unstable version and I noticed that the pantheon is missing in the build menu. Is it being changed? Just want to make sure I haven't messed up my assets while switching versions.

**Reply by MSTVD** [reply]
Hey. Are you checking in the "Grand Temple" category?

**Reply by P2C2E1993** [reply]
Yes, but I can't see it in the Grand Temple menu. I can only see the Grand temples to the 5 gods.

**Reply by crudelios** [reply]
That can happen if the map doesn't provide the required resources to build the pantheon. Most likely you don't have access to gold.

**Reply by P2C2E1993** [reply]
Ah that's sad. I'm playing on the Valentia map from the scenario, and it doesn't have iron, which means I can't build the mint to get gold either. I think you would need to modify many scenario maps to accomodate the new resources.

**Reply by crudelios** [reply]
You are correct. I did try to retrofit the main campaign maps to support the new resources, but I didn't get every single case unfortunately.

There's a cheat code that unlocks every resource so you can use that instead.

**Comment by Ouaz** [comment]
> Update french.c
> Update German translation
> Update Portuguese translation
> Update russian.c

> -> I am wondering when would be the next official release. Any time soon?
> -> Don't expect it for the next six months up to a year.

"_It was a trap._" :amiral_ackbar:

**Reply by crudelios** [reply]
Haha but the amount of new features will be amazing, it will give some needed freshness to the game!
